packages/web-twig/src/Resources/components/Link/Link.twig

```diff
@@ -4,17 +4,30 @@
 {%- set _href = props.href -%}
 {%- set _isDisabled = props.isDisabled | default(false) -%}
 {%- set _isUnderlined = props.isUnderlined | default(false) -%}
+{%- set _underline = props.underline | default('hover') -%}
```

Comment:
why is in twig default hover and in react there is undefined?

Reply:
should be hover, will fix it in react